packages/web-components/fast-foundation/src/date-picker/date-picker.ts

```diff
@@ -0,0 +1,1262 @@
+import {
+    attr,
+    booleanConverter,
+    DOM,
+    observable,
+    SyntheticViewTemplate,
+} from "@microsoft/fast-element";
+import {
+    keyArrowDown,
+    keyArrowLeft,
+    keyArrowRight,
+    keyArrowUp,
+    keyEnter,
+    keyEscape,
+    keySpace,
+} from "@microsoft/fast-web-utilities";
+import type {
+    FoundationElementDefinition,
+    FoundationElementTemplate,
+} from "../foundation-element";
+import type { ListboxElement } from "../listbox";
+import {
+    DateFormatter,
+    DayFormat,
+    MonthFormat,
+    WeekdayFormat,
+    YearFormat,
+} from "../calendar/date-formatter";
+import type { TextField } from "../text-field";
+import { FormAssociatedDatePicker } from "./date-picker.form-associated";
+
+/**
+ * Date picker options
+ * @alpha
+ */
+export type DatePickerOptions = FoundationElementDefinition & {
```

Comment:
Can these icons be overridden for each instance?  Ie. will all "fast-date-picker" instances have the same icons.

Reply:
"The previous and next icons can only be changed for all."  Should that be customizable per instance?  I suppose that would be an easy enhancement if folks wanted it.